app/src/main/java/com/fueledbycaffeine/mivote/MainActivity.kt

```diff
@@ -3,9 +3,11 @@ package com.fueledbycaffeine.mivote
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import androidx.compose.foundation.Text
+import androidx.compose.foundation.layout.Column
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Surface
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
```

Comment:
Pretty sure this will fail lint now for unused import 🤪

Reply:
i have NO idea how that got in there lmao